main_test.go

```diff
@@ -99,9 +99,33 @@ func setupPOSIXEnv(env *testscript.Env) error {
 	})
 }
 
+// Set up the testing environment for Windows.  Works the same as on POSIX with
```

Comment:
nit: Standard Go doc comments begin with the function name, so this should be:

```go
// setupWindowsEnv sets up the testing environment for Windows. Works the same as...
```

Reply:
Fixed. You can tell I'm not usually a Go programmer 😄